app/encoding.go

```diff
@@ -1,45 +1,17 @@
 package app
 
 import (
-	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/codec"
-	"github.com/cosmos/cosmos-sdk/codec/types"
-	"github.com/cosmos/cosmos-sdk/std"
-	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+	"github.com/NibiruChain/nibiru/app/codec"
 )
 
 // EncodingConfig specifies the concrete encoding types to use for a given app.
 // This is provided for compatibility between protobuf and amino implementations.
-type EncodingConfig struct {
-	InterfaceRegistry types.InterfaceRegistry
-	Marshaler         codec.Codec
-	TxConfig          client.TxConfig
-	Amino             *codec.LegacyAmino
-}
+type EncodingConfig = codec.EncodingConfig
 
 // MakeEncodingConfig creates an EncodingConfig for an amino based test configuration.
 func MakeEncodingConfig() EncodingConfig {
```

Comment:
you can call this RegisterEncodingModules, or something like that.

Reply:
I was trying to prevent changing it everywhere else but leaving in the ability to import the `codec.Codec` without the app. Are you suggesting to rename the `MakeEncodingConfig` function?

Reply:
I think you can use everywhere the one you moved into codec.MakeEncodingConfig() and remove this one

Reply:
If u dont want to avoid changing it everywhere, you can try if it works the aliasing, ase as you did in line 9 but for the function from codec.

Reply:
Roger. Will refactor into the way you've suggested

Reply:
Addressed by 253cb65234b0e3f8fc703e45b189cf5ae2290298

Note that the registration part is required in most of our tests because it's used for our `x` modules and some of the native ones like distribution.